### DISCUSSION TITLE
Changing recipient attribute to other name

### DISCUSSION THREAD

**Discussion by sunzoje** [excid3/noticed#431] [Q&A] [answered]
In https://github.com/excid3/noticed/blob/3333697c0e7c2c68571d8699c6bc5a7fc26a18bb/db/migrate/20231215190233_create_noticed_tables.rb#L19 changing *recipient* to *receiver* will create table with *receiver_type* attribute in *noticed_notifications* table.
I'm getting,
```
     ActiveModel::UnknownAttributeError:
       unknown attribute 'recipient_type' for Noticed::Notification.
```
error, when calling notifier's *deliver* method.

**Comment by excid3** [comment, marked as answer]
You'll have to change _all_ references of `recipient` in the code.
